modules/tile-converter/test/i3s-converter/i3s-converter.spec.js

```diff
@@ -348,3 +350,28 @@ test('tile-converter - Converters#layer json should contain fullExtent field', a
   await cleanUpPath('data/BatchedTextured');
   t.end();
 });
+
+test('tile-converter - Converters#TRIANGLE_STRIPS should be converted to independent TRIANGLES', async (t) => {
```

Comment:
Make a geometry test in `modules/tile-converter/test/i3s-converter/helpers/geometry-converter.spec.js`. The only thing we need to check is geometry content.
Whole converter tests are noisy and slower